.github/workflows/ci.yaml

```
@@ -34,7 +34,8 @@ jobs:
           export URL=$(curl -s https://api.github.com/repos/ethersphere/bee-local/releases/latest | jq -r .tarball_url)
           curl -Ls ${URL} -o bee-local.tar.gz
           tar --strip-components=1 --wildcards -xzf bee-local.tar.gz ethersphere-bee-local-*/{beeinfra.sh,helm-values,hack}
-          sed -i 's/IMAGE_TAG="latest"/IMAGE_TAG="0.4.2"/' beeinfra.sh
+          # TODO remove comment and update version number once the release is out
+          # sed -i 's/IMAGE_TAG="latest"/IMAGE_TAG="0.4.2"/' beeinfra.sh
```

Just so we don't forget about this :)
```suggestion
          sed -i 's/IMAGE_TAG="latest"/IMAGE_TAG="0.5.0"/' beeinfra.sh
```